kotlinx-coroutines-debug/src/DebugProbes.kt

```diff
@@ -65,12 +65,11 @@ public object DebugProbes {
 
     /**
      * Whether coroutine creation stack traces should be captured.
-     * When enabled, for each created coroutine a stack trace of the current
-     * thread is captured and attached to the coroutine.
+     * When enabled, for each created coroutine a stack trace of the current thread is captured and attached to the coroutine.
      * This option can be useful during local debug sessions, but is recommended
-     * to be disabled in production environments to avoid stack trace dumping overhead.
+     * to be disabled in production environments to avoid performance overhead of keeping debug probes enabled.
```

Comment:
I understood the previous version better. With `enableCreationStackTraces`, you copy a bunch of stackframe descriptions with many strings around *on every coroutine creation*, which is expensive, ok, this makes sense. What does the new text mean? This option changes the behavior *while* the debug probes are enabled, it doesn't force them to be enabled longer than needed, right?